sycl/include/sycl/ext/oneapi/experimental/chunk.hpp

```diff
@@ -26,18 +26,18 @@ namespace sycl {
 inline namespace _V1 {
 namespace ext::oneapi::experimental {
 
-template <size_t PartitionSize, typename ParentGroup> class fixed_size_group;
+template <size_t PartitionSize, typename ParentGroup> class chunk;
 
 template <size_t PartitionSize, typename Group>
 #ifdef __SYCL_DEVICE_ONLY__
-[[__sycl_detail__::__uses_aspects__(sycl::aspect::ext_oneapi_fixed_size_group)]]
+[[__sycl_detail__::__uses_aspects__(sycl::aspect::ext_oneapi_chunk)]]
 #endif
 inline std::enable_if_t<sycl::is_group_v<std::decay_t<Group>> &&
                             std::is_same_v<Group, sycl::sub_group>,
-                        fixed_size_group<PartitionSize, Group>>
-get_fixed_size_group(Group group);
+                        chunk<PartitionSize, Group>>
+get_chunk(Group group);
```

Comment:
[`sycl_ext_oneapi_non_uniform_groups`](https://github.com/intel/llvm/blob/sycl/sycl/doc/extensions/experimental/sycl_ext_oneapi_non_uniform_groups.asciidoc) does not define `get_chunk`, I suppose you meant `chunked_partition`